includes/Shopping/Shopify_Query.php

```diff
@@ -151,16 +151,17 @@ protected function execute_query( string $query ) {
 	 * @since 1.21.0
 	 *
 	 * @param string $search_term Search term to filter products by.
+	 * @param string $orderby Sort collection by product attribute.
+	 * @param string $order Order sort attribute ascending or descending.
 	 * @return string The assembled GraphQL query.
 	 */
-	protected function get_products_query( string $search_term ): string {
+	protected function get_products_query( string $search_term, string $orderby, string $order ): string {
 		$search_string = empty( $search_term ) ? '*' : '*' . $search_term . '*';
-
-		// TODO(#11154): Support different sortKeys.
-		// Maybe use "available_for_sale:true AND " query to only show items in stock.
+		$sortkey       = 'date' === $orderby ? 'CREATED_AT' : strtoupper( $orderby );
+		$reverse       = 'asc' === $order ? 'false' : 'true';
 		return <<<QUERY
 {
-  products(first: 100, sortKey: CREATED_AT, query: "title:$search_string") {
+  products(first: 100, sortKey: $sortkey, reverse: $reverse, query: "title:$search_string") {
```

Comment:
I am not sure this is correct. Instead of using `reverse`, shouldn't this is last?

https://shopify.dev/api/usage/pagination-graphql#backward-pagination

Reply:
I think `reverse` is the right property to use for `ASC` / `DESC` type of queries
see - https://youtu.be/S37WsC8GzSA?t=352

Reply:
@spacedmonkey That documentation is for pagination, while this PR is for sorting.